broker-daemon/block-order-worker/index.spec.js

```diff
@@ -202,45 +203,12 @@ describe('BlockOrderWorker', () => {
       expect(Order.fromStorage).to.have.been.calledWith(fakeKey, fakeValue)
       expect(filtered).to.be.equal(false)
     })
-
-    it('works a block order when one is created', async () => {
-      const fakeBlockOrder = 'my fake'
-      eventsOn.withArgs('BlockOrder:create').callsFake(async (evt, fn) => {
-        await delay(10)
-        fn(fakeBlockOrder)
-      })
-      worker = new BlockOrderWorker({ orderbooks, store, logger, relayer, engines })
-      worker.workBlockOrder = sinon.stub().resolves()
-
-      await delay(15)
-
-      expect(worker.workBlockOrder).to.have.been.calledOnce()
-      expect(worker.workBlockOrder).to.have.been.calledWith(fakeBlockOrder)
-    })
-
-    it('fails a block order if working it fails', async () => {
```

Comment:
Same as the above test. This test was removed from this section of the spec because we no longer use events in the initialization of a BlockOrderWorker. we test for this below.